Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# 88 holy order names

## Changelog:
- Added holy orders names #88 


## Developer changelog:
- Edited all religion files in common\religion\religions, added holy order names for all religions.
- Description added to localization\english\wc_holy_orders_l_english.txt
- Deleted file of same name as above (wc_holy_orders_l_english.txt), only had 3 holy order names.


# How to test:
- Activate debug mode (if you have steam, go to the game in the steam app, right-click, properties, general, add -debug_mode to launch options), start the game somewhere and go to decisions, you will find a decision to test holy orders (it creates 3 holy orders for each faith). Use that decision and you can go hoping on a few characters to see the religion names and decide if you like them:

![image](https://user-images.githubusercontent.com/86002099/126082061-5b5dd25c-0b6e-4444-8c3f-ebe2d523fd9c.png)
